### PR TITLE
Update Portuguese Brazilian Translation

### PR DESCRIPTION
**Pull request by altiereslima**

### CODE REVIEW

File: intl/msg_hash_pt_br.c [hunk 1]
```diff
@@ -27,14 +27,6 @@
 #ifdef RARCH_INTERNAL
 #include "../configuration.h"
 
-#if defined(_MSC_VER) && !defined(_XBOX) && (_MSC_VER >= 1500 && _MSC_VER < 1900)
```
Comment (bparker06) [inline intl/msg_hash_pt_br.c]: Why was this removed?

Comment (altiereslima) [reply intl/msg_hash_pt_br.c]: image

Comment (altiereslima) [reply intl/msg_hash_pt_br.c]: based on english msg_hash

Comment (LazyBumHorse) [reply intl/msg_hash_pt_br.c]: According to the [VC++ docs](https://docs.microsoft.com/en-us/cpp/preprocessor/execution-character-set?view=vs-2019) the removed pragmas _might_ not be required, because msg_hash_pt_br does start with a (UTF-8) BOM and MSVC should thus use UTF-8 as the execution character set, though the documentation isn't super clear about that.

In the end it is much safer to not assume but explicitly tell (MSVC < 2015 Update 2) to use UTF-8 encoding for the strings, so the removed pragmas should definitely be restored.

The english msg_hash does not have the `#pragma execution_character_set("utf-8")` because it is only using ASCII strings, which have the same encoding in virtually any code page.
